From 79b86db65ae07aa91a45406c07c614b61fb488cd Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Thu, 02 Dec 2021 14:15:53 +0200
Subject: [PATCH] [SW-0] gaudi2: security: remove __unused attribute

Removed __unused attribute for hl_secure_hbw_rr() as
function is used.

Change-Id: I1da4554e95c63068671224ef5fab2d15e1bb42da
Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
---

diff --git a/arch/gaudi2/common/security.c b/arch/gaudi2/common/security.c
index 5d76eff..39ec6f3 100644
--- a/arch/gaudi2/common/security.c
+++ b/arch/gaudi2/common/security.c
@@ -621,7 +621,7 @@
 		hl_secure_update_rr_reg_hbw(rr_idx, cfg->instances_dist * i, &rr_reg);
 }
 
-__unused static void hl_secure_hbw_rr(int rr_idx, u64 min, u64 max, struct hl_rr_eng_select rr_sel)
+static void hl_secure_hbw_rr(int rr_idx, u64 min, u64 max, struct hl_rr_eng_select rr_sel)
 {
 	int i;
 
